### DISCUSSION TITLE
can markmap-cli create html with offline data?

### DISCUSSION THREAD

**Discussion by lizhihome** [markmap/markmap#233] [General]
when i use npx markmap-cli mw.md, i defined   embedAssets: true in the .md, but mw.html cannot have offline data. is there any option to create offline mw.html?
Thanks,

**Comment by gera2ld** [comment]
`npx markmap-cli --offline mw.md`

**Comment by lizhihome** [comment]
yes, it' works, thanks. and it's better to add this option to manual.
